src/isIsogram.test.js

```diff
@@ -6,4 +6,28 @@ describe('isIsogram', () => {
   it(`should be declared`, () => {
     expect(isIsogram).toBeInstanceOf(Function);
   });
+
+  it(`should return 'true' when argument is ''`, () => {
+    const result = isIsogram('');
+
+    expect(result).toEqual(true);
+  });
+
+  it(`should return 'false' when argument is 'look'`, () => {
+    const result = isIsogram('look');
+
+    expect(result).toEqual(false);
+  });
+
+  it(`should be case-insensitive'`, () => {
```

Comment:
The description of this test case has a minor typo. It should be `should be case-insensitive` instead of `should be case-insensitive'`. The single quote at the end seems to be misplaced.